docs/development/visualize/development-visualize-index.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[development-visualize-index]]
+== Developing Visualizations
+
+Easiest way to add new visualizations to kibana or to show existing kibana visualizations in your plugin is using 
+Visualize Plugin. This part of documentation is split into two parts. First part will tell you all you need to know to
+use existing kibana visualization in your plugin and second step will focus on creating your own custom visualization.
```

Comment:
Consider slight rewrite wrt language:

`Kibana Visualizations are the easiest way to add additional functionality to Kibana. This part of documentation is split into two parts. The first part tells you all you need to know on how to
embed existing Kibana Visualizations in your plugin. The second step explains how to create your own custom visualization.`

Reply:
I'd add a short hedge. This is not public API (yet).

> These pages document internal APIs and are not guaranteed to be supported across future versions of Kibana. However, these docs will be kept up-to-date to reflect the current implementation of Visualization plugins in Kibana.